### ISSUE TITLE
SSH connection

### ISSUE THREAD

**Issue by saqibomer** [vapor/mysql-kit#316]
I could not find any method for connecting via SSH. Is there any planned enhancement for supporting ssh connection?

**Comment by 0xTim** [comment]
I don't think this is planned. You have a couple of options, create the tunnel outside of the application and connect normally, or use [SwiftNIO. SSH](https://github.com/apple/swift-nio-ssh) to create a tunnel in your application and connect via that
